azurePipeline.yml

```
@@ -0,0 +1,104 @@
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'vs2017-win2016'
```

```suggestion
    vmImage: $(imageName)
```

Consider adding support for OSX as well as Windows. https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops#add-additional-platforms

---

Not originally part of the spec, but easy change :)